inst/templates/get-started.Rmd

```diff
@@ -383,6 +386,57 @@ vroom::vroom_write(to_edit, "to_edit.csv", delim = ",")
 writexl::write_xlsx(to_edit, "to_edit.xlsx")
 ```
 
+### Calculate coverage statistics
+
+```{r}
+suggested <- to_edit %>%
+  filter(suggest_match)
+
+#### Number of matched companies based on suggested matches
+matched_rows <- suggested %>% nrow()
+matched_rows
+
+#### Share of matched companies from total loanbook companies
+total_rows <- loanbook %>%
+  distinct(company_name) %>%
+  nrow()
+matched_share <- matched_rows / total_rows
+matched_share
+```
+
+#### Number and share of matched companies classified by misc_info from total loanbook companies
+
+misc_info should be replaced with variables like sectors, headcount etc.
+
+```{r}
+x <- loanbook %>% count(misc_info)
+y <- suggested %>% count(misc_info)
+misc_share <-
+  left_join(x, y, by = c("misc_info"), suffix = c("_loanbook", "_merged")) %>%
+  mutate(n_share = n_merged / n_loanbook)
+```
+
+#### Calculate loan exposure categorised based on misc_info 
+
+id_tilt column is used as the substitute due to unavailability of loan amount
+column in sample data. Please replace id_tilt column with the loan amount column.
```

Comment:
More importantly, how about creating a new `loan_amount` column with artificial values? The example would make more sense and would be easier for the users as they would not need to edit the code.

Reply:
@maurolepore I have added a new sample column called 'loan_amount' with artificial values to avoid any confusion.

Reply:
@maurolepore Can you please explain which suggestions you are talking about in the below comment after my change?
Here: Why are we calculating coverage on the data before the user accepts the suggestions? What if all suggestions are wrong? Would the coverage be still useful?

Reply:
Sure. 

In this PR, coverage is calculated based on the dataset `to_edit` -- not on the dataset `edited`. That means coverage is calculated before the user had the chance to validate the matching suggestions by setting  `accept_match = TRUE` using s spreadsheet editor.

To understand the potential problem, consider the extreeme case where none of the suggested matches correspond to the same company -- i.e. consider the case where all suggestions are false positives. How meaningful is coverage? I think the answer is not meaningful at all.

In contrast, see how r2dii.match calculates coverage:
https://rmi-pacta.github.io/r2dii.match/articles/matching-coverage.html[](https://rmi-pacta.github.io/r2dii.match/articles/matching-coverage.html
That article) calculates coverage after calling `prioritize()` -- [which assumes the user already validated the matches manually](https://rmi-pacta.github.io/r2dii.match/index.html#id_2-prioritize-validated-matches).

This seems to be a pretty big conceptual issue. And I may be getting it all wrong. But please make sure to discuss this with Tilman and let me know what you decide.

Reply:
@maurolepore I have consulted about your comment with Tilman. And he also agrees that coverage should be calculated based on accepted matches and not suggested matches. Therefore, I have made the required change in the code. Please have a final look at the code and let me know if any other change is needed! Thanks for your feedback!

Reply:
@maurolepore Please let me know if the prioritize function has any additional functionality in comparison to my current solution, so that I can also apply it in my usecase.

Reply:
I wouldn't worry about `prioritize()` here. As you can see in it's [helpfile](https://rmi-pacta.github.io/r2dii.match/reference/prioritize.html), `prioritize()` is unrelated to coverage.